packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from '@vtex/store-ui'
+import React, { FC } from 'react'
+
+type Props = {
+  variant?: string
+  as: React.ElementType
+  width?: number
+  height?: number
+  src: string
+  alt: string
+  loading?: string
+}
+
+export const HeaderMinicartDrawerContentImage: FC<Props> = ({
+  loading = 'lazy',
+  ...props
+}) => {
+  const imageProps = {
+    loading,
+    ...props,
+  }
+
+  return (
+    <Box variant={props.variant}>
+      <Box {...imageProps} />
```

Comment:
Do this box has the same variant from the box above ?

Reply:
this is a mistake, I will fix it!